### DISCUSSION TITLE
Enahancement for supporting Options?

### DISCUSSION THREAD

**Discussion by NagarajuGunda** [gbeced/basana#14] [Ideas]
Hi,

Is there any plan to support option contracts?

**Comment by gbeced** [comment]
Not at the moment.

**Reply by NagarajuGunda** [reply]
I've been using pyalgotrade and written my logic to support option trading on top of it.

Now, I want to move to async based solution and exploring basana

Do you have anything in mind on how to approach and implement OptionContract or any other contract (Stock/Futures) as basana currently supports bs.Pair based approach

**Reply by gbeced** [reply]
No, and by the way if you're going to copy my code at least leave the copyright notice:

* https://github.com/NagarajuGunda/PyAlgoMate/blob/develop/pyalgomate/barfeed/resampled.py
* https://github.com/gbeced/pyalgotrade/blob/master/pyalgotrade/barfeed/resampled.py
